debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/strategy/mariadb/MariaDbGtidSet.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql.strategy.mariadb;
+
+import java.util.function.Predicate;
+
+import io.debezium.DebeziumException;
+import io.debezium.connector.mysql.GtidSet;
+
+/**
+ * @author Chris Cranford
+ */
+public class MariaDbGtidSet implements GtidSet {
+
+    public MariaDbGtidSet(String gtid) {
+    }
+
+    @Override
+    public boolean isEmpty() {
+        throw new DebeziumException("NYI");
```

Comment:
If these exceptions are going to stay for a while then I'd recommend to use `UnsupportedException("Not yet implemented")` for the reader's advanatge. If it is only placeholder for the sake of this PR then it can be kept as is now.

Reply:
No, these are only for my benefit. These will all be removed once I finalize the implementation.